### ISSUE TITLE
Deployment option icons are not arranged horizontally

### ISSUE THREAD

**Issue by pradeeka7** [TheThingsIndustries/lorawan-stack-docs#1168]
#### Summary

Deployment option icons are not arranged horizontally. Instead, they arranged vertically. For example, https://www.thethingsindustries.com/docs/the-things-stack/packet-broker/configure/

<img src="https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/36432168/bca1e4af-17c1-4f3d-a470-2dd6dd36fd28" width="200">
<p>
<img src="https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/36432168/bd59037b-ff16-4fd7-8350-bc265349df88" width="200">

#### Why do we need this ?

The vertical arrangement of icons occupies more space and results in a cluttered page layout.

#### What is already there? What do you see now?

...

#### What is missing? What do you want to see?

Fix the layout of the document to arrange icons horizontally.

#### How do you propose to document this?

...

#### Can you do this yourself and submit a Pull Request?

No, I think @kschiffer can take this over.

**Comment by nejraselimovic** [comment]
hey @kschiffer can you arrange this to be fixed any time soon?

**Comment by nejraselimovic** [comment]
this looks better now but I still think it looks weird, I was expecting for these icons to be in line with text and not above, @kschiffer is this how it is supposed to look?
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/65215579/d63132f8-ac68-4fc4-8124-dd32804099fa)

**Comment by KrishnaIyer** [comment]
Yeah we'll revisit these icons at some point. I think we should keep this for now.
